crates/iceberg-catalog/src/implementations/postgres/user.rs

```diff
@@ -0,0 +1,93 @@
+use super::dbutils::DBErrorHandler;
+use crate::api::management::v1::{User, UserOrigin};
+use crate::service::{ErrorModel, Result};
+use http::StatusCode;
+use sqlx::PgConnection;
+use uuid::Uuid;
+
+#[expect(dead_code)]
+pub(crate) async fn get_user(user_id: Uuid, connection: &mut PgConnection) -> Result<User> {
```

Comment:
We need search additionally for the UI - and also a List so that users can be deleted if they no longer want to be searchable